lib/sarif/bundle_audit_sarif.rb

```diff
@@ -29,6 +29,7 @@ def parse_issue(issue)
       if issue[:line_number]
         result[:start_line] = issue[:line_number]
         result[:start_column] = 1
+        result[:code] = issue[:name].to_s
```

Comment:
Can you include the comment present in your PR description here in the code for readability?
# Code stores the gem name without version in snippet.

Reply:
added, great suggestion!